pinot-spi/src/main/java/org/apache/pinot/spi/annotations/ScalarFunction.java

```diff
@@ -46,5 +46,10 @@
 
   boolean enabled() default true;
 
-  String name() default "";
+  // A comma-separated function names to register in FunctionRegistry
+  String names() default "";
+
+  String alias() default "";
```

Comment:
Can this return a `List<String>`?

Reply:
if we want to have a default value, then it has to be a constant. Otherwise, all existing scalar functions have to set the annotation value.

Reply:
You can use an array instead:

```java
String[] names() default {};
```